docs/modules/device-lab-management/pages/ios-devices/index.adoc

```diff
@@ -1,8 +1,18 @@
 = iOS/iPadOS devices
+:page-aliases: devices:local-devices:configure-ios-voiceover.adoc
+
 :navtitle: iOS/iPadOS devices
 
 Learn how to host and manage iOS/iPadOS devices in Kobiton.
 
+[NOTE]
+.Limitations
+====
+
+* Perform Voiceover is no longer supported due to a change in iOS's Bluetooth protocol. We will let you know when this feature is supported again.
```

Comment:
Capitalize it VoiceOver instead of Voiceover per https://www.apple.com/voiceover/info/guide/_1121.html